### PR TITLE
fix(notion):Notion integration

### PR DESCRIPTION
**Pull request by ivillacreative**
Please remember the [Contributing Guidelines](https://github.com/toggl/toggl-button/blob/master/.github/CONTRIBUTING.md) :heart:

## :star2: What does this PR do?

* Adds Toggl button on notion pages

## :bug: Recommendations for testing

All changes should be tested across Chrome and Firefox.

## :memo: Links to relevant issues or information

#1454

### CODE REVIEW

File: src/scripts/content/notion.js [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
       className: 'notion',
-      description: descriptionElem.textContent.trim(),
-      projectName: projectElem && projectElem.textContent,
-      calculateTotal: true,
-      buttonType: 'minimal'
+      description: descriptionElem
     });
 
     container.appendChild(link);
+    console.log(togglButtonLoc.parentNode);
```
Comment (karlhorky) [inline src/scripts/content/notion.js]: Probably want to remove this, yeah?

Comment (ivillacreative) [reply src/scripts/content/notion.js]: oops, yea I should lol

**Comment by karlhorky** [comment]
cc @tcrammond seems like this fixes the Notion implementation from #1203. Didn't get a chance to test it yet.

**Comment by tcrammond** [comment]
Hey @ivillacreative, it's been a little while since this PR was opened.

Do you still have plans to continue it, or would you like someone else to pick it up?

**Comment by lise-toggl** [comment]
Superceded by #1612 and #1617

**Pull request closed without merging**